CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ if (APPLE)
     option(WHISPER_METAL_NDEBUG          "whisper: disable Metal debugging"      OFF)
     option(WHISPER_COREML                "whisper: enable Core ML framework"     OFF)
     option(WHISPER_COREML_ALLOW_FALLBACK "whisper: allow non-CoreML fallback"    OFF)
+    option(WHISPER_EMBED_METAL_LIBRARY   "whisper: embed Metal library"          OFF)
```

Comment:
Rename to `WHISPER_METAL_EMBED_LIBRARY`

Reply:
It looks like it is not possible to embed this approach into Objective-C source, however if  the aim is to avoid having another file in the source tree, I updated the build process so that the assembly is autogenerated during the build phase.